src/languages/swift.js

```diff
@@ -96,7 +82,114 @@ export default function(hljs) {
         { begin: /\b0b([01]_*)+\b/ },
       ]
   };
-  SUBST.contains = [NUMBER];
+
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#grammar_string-literal
+  var ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\[0\\tnr"']|\\u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var RAW1_ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\#[0\\tnr"']|\\#u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var RAW2_ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\##[0\\tnr"']|\\##u\{[0-9a-fA-F]{1,8}\}/
```

Comment:
Could we use `variants` to make these entirely different syntaxes a little more visible?